engine/class_modules/sc_mage.cpp

```diff
@@ -8587,9 +8620,9 @@ void mage_t::create_buffs()
 
 
   // Set Bonuses
-  buffs.intuition = make_buff( this, "intuition", find_spell( 455681 ) )
+  buffs.intuition = make_buff( this, "intuition", find_spell( 1223797 ) )
                       ->set_default_value_from_effect( 1 )
-                      ->set_chance( sets->set( MAGE_ARCANE, TWW1, B4 )->effectN( 1 ).percent() );
+                      ->set_chance( find_spell( 1223798 )->effectN( 1 ).percent() );
```

Comment:
This is missing the talent check. It just triggers regardless of whether you have the talent or not.